src/modules/launcher/Plugins/Microsoft.Plugin.Sys/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+    "ID":"CEA08895D2544B019B2E9C5009600DF4",
+    "ActionKeyword":"*",
+    "Name":"System Commands",
+    "Description":"Provide System related commands. e.g. shutdown,lock,setting etc.",
+    "Author":"Wox",
+    "Version":"1.0.0",
+    "Language":"csharp",
+    "Website":"https://aka.ms/powertoys",
+    "ExecuteFileName":"Microsoft.Plugin.Sys.dll",
+    "IcoPath":"Images\\lock.png"
```

Comment:
> @benjamhooper do you still need a plugin icon? Like the other plugins have as well?
> 
> If so, I'd go for the power button icon to represent the plugin itself and the actual action.

Assuming you'd just change this to the power button icon, dark or light?

Reply:
Yep! Other plug-ins use .dark.

For now now it doesn't really matter, because I don't think it's shown anywhere.. but maybe in the future if we have plug-in management capabilities in Settings it would be nice to show.

Reply:
Gotcha, hope this will come to fruition: https://github.com/microsoft/PowerToys/issues/5273 :)